Layout map item overviews position can be tweaked (#4728)
qgis/QGIS-Documentation: source/docs/user_manual/print_composer/composer_items/composer_map.rst

* Rename it with a double-click;
* With the |signPlus| and |signMinus| buttons, add or remove overviews;
* With the |arrowUp| and |arrowDown| buttons, move up and down an overview in
  the list, hence move it on top or bottom of another one, within the map item.

Then select the overview item in the list and check the |checkbox|
:guilabel:`Draw "<name_overview>" overview` to enable the overview
drawing over the selected map frame. You can customize it with:

* The :guilabel:`Map frame` selects the map item whose
  extents will be drawn on the present map item.
* The :guilabel:`Frame Style` uses the :ref:`symbol properties
  <symbol-selector>` to render the overview frame.
* The :guilabel:`Blending mode` allows you to set different transparency blend
  modes.
* The |checkbox| :guilabel:`Invert overview` creates a mask around the extents
  when activated: the referenced map extents are shown clearly, whereas
  everything else is blended with the frame fill color.
* The |checkbox| :guilabel:`Center on overview` pans the map item content so

>   that the overview frame is displayed at the center of the map. You can only

Reviewer: ```suggestion
  that the overview frame is displayed at the center of the map.
  You can select this for as many overview maps as you like, but
```

>   activate one overview item to center, when you have several overviews.

Reviewer: ```suggestion
  the centering will be according to the last one.
```

Author: Indeed, but this behavior is a bug IMO (https://github.com/qgis/QGIS/issues/33597): there's no interest in checking more than one (only the last wins and unchecking the last does not center on the penultimate so useless). And I'm not sure we should document buggy behavior

* The :guilabel:`Position` controls exactly where in the map item's layer stack

>   the overview will be placed, e.g. allowing them to draw a overview

Reviewer: ```suggestion
  the overview will be placed, e.g. allowing an overview extent to be
```

>   extent below some feature layers such as roads whilst drawing it

Reviewer: ```suggestion
  drawn below some feature layers such as roads whilst drawing it
```

  above other background layers. Available options are:

  * :guilabel:`Below map`
  * :guilabel:`Below map layer` and :guilabel:`Above map layer`, selecting in
    :guilabel:`Stacking layer` the layer to consider
  * :guilabel:`Below map labels` and :guilabel:`Above map labels`: since

>     labels are drawn over all the features, the overview is placed respectively

Reviewer: Not sure I follow / understand this explanation, but then I am not too familiar with this.

Author: (up to QGIS 3.10 but i'm not sure it has changed - i did not yet check the mask feature impact) when a map is rendered in QGIS, geometries of features are processed first. When all the layers have their features rendered, then their labels are rendered, so labels are always placed above geometry.
In this case, "below map labels" means below map labels but above map layers, and the second one means above everything in the map item. If it's clearer and you have a suggestion of rewording...

>     labels are drawn over all the features, the overview is placed respectively

Reviewer: ```suggestion
    labels are drawn above all the other map features, for these options
```

>     above all the map layers or above the map item.

Reviewer: ```suggestion
    the overview is placed above all map contents except for
    the labels (if ``Below map labels`` chosen).
    For ``Above map labels``, the overview is placed above
    the labels too.
```
